README.rst

```diff
@@ -845,6 +845,55 @@ wait_for_motion(timeout_secs=10, consecutive_frames=None, noise_threshold=None,
       to search for motion. White pixels select the area to search; black
       pixels the area to ignore.
 
+wait_until(callable_, timeout_secs=10, interval_secs=0)
+    Run a function or lambda until it succeeds, or until a timeout.
```

Comment:
> Perhaps "Wait until a condition becomes True or timeout".

Done (pushed a fixup commit). I used a lowercase "true" because it works with any truthy value, not just `True`.